core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/output/GeoWaveOutputFormat.java

```diff
@@ -281,7 +284,7 @@ public OutputCommitter getOutputCommitter(
 	protected static class GeoWaveRecordWriter extends
 			RecordWriter<GeoWaveOutputKey, Object>
 	{
-		private final Map<ByteArrayId, IndexWriter> indexWriterCache = new HashMap<ByteArrayId, IndexWriter>();
+		private final Map<ByteArrayId, IndexWriter> adapterWriterCache = new HashMap<ByteArrayId, IndexWriter>();
```

Comment:
we are still calling them index writers - I could understand changing it to adapter writer, but that could be up for debate.  This one isn't a big deal, just a variable name, but its seems inconsistent, and worth questioning "do we want to call these writers adapter writers?"